### DISCUSSION TITLE
Defined volumes dont make folders in Windows Docker Desktop

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2196] [Questions] [answered]
volumes:
  nextcloud_aio_mastercontainer: #works - creates folder automatically
    name: nextcloud_aio_mastercontainer
    driver: local
    driver_opts:
      o: bind
      type: none
      device: /dockerdata/ncmc
  nextcloud_aio_nextcloud_datadir: # does not create folder automatically :-(
    name: nextcloud_aio_nextcloud_datadir
    driver: local
    driver_opts:
      o: bind
      type: none
      device: /dockerdata/nextcloud

**Comment by szaimen** [comment, marked as answer]
Hi, did you see this? https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir

![image](https://user-images.githubusercontent.com/42591237/227707705-6c88e5cf-ebb1-4cc2-a676-1c084fc263fb.png)

**Reply by FlattusBlastus** [reply]
Of course. I am asking you to make docker compose installs work reliably.

**Reply by szaimen** [reply]
It works reliably on my end

**Reply by FlattusBlastus** [reply]
_Without doing this first? In compose, the other defined volumes as folders _dont_ get created.

docker volume create ^
--driver local ^
--name nextcloud_aio_nextcloud_datadir ^
-o device="/d/dockerdata/nextcloud" ^
-o type="none" ^
-o o="bind"

docker volume create ^
--driver local ^
--name nextcloud_aio_backupdir ^
-o device="/d/dockerbackup" ^
-o type="none" ^
-o o="bind"

Wouldn't it be great to eliminate this step?
